tests/modules/submission/resources/test_ensure_submission.py

```diff
@@ -33,7 +33,7 @@ def test_ensure_clone_submission_by_uuid(
             '4d46c55d-accf-29f1-abe7-a24839ec1b95'
         )
 
-        assert clone.temp_submission.commit_houston_api_version == '0.1.0.8b208226'
+        # assert clone.temp_submission.commit_houston_api_version == '0.1.0.8b208226'
```

Comment:
Yes. I wanted to talk about this one, this was specifically not something I added, it was already in the test. how do you think we should manage what the required houston_api_version should be in future

Reply:
I found the bug for this test and fixed it in commit dddea84